_posts/2017-09-15-puppet-extension-deep-dive-part1.md

```diff
@@ -13,14 +13,14 @@ tags:
   - language
   - server
   - extension
-modified: 2018-01-02
+modified: 2018-06-08
 ---
 
 [James Pogran](https://github.com/jpogran) and I have been busy writing new features and fixing bugs in the Puppet VS Code extension.  So for the next series of blog posts I am going to dive deep into the source code for the extension and explain how parts of it work and how to setup a development environment for the extension.
 
 But first some handy links:
 
-[Source Code](https://github.com/jpogran/puppet-vscode)
+[Source Code](https://github.com/lingua-pupuli)
```

Comment:
Suggest link and those immediately after be in an unordered list.

Reply:
Formatting > Semantics